index.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="utf8" />
-    <title>your project name</title>
+    <title>Quiz and fun</title>
```

Comment:
Should we rename it as our team name, `Quiz Your Future`?

Reply:
Yes, sure!